event_registration_partner_unique/models/event.py

```diff
@@ -42,6 +57,8 @@ def _check_forbid_duplicates(self):
                         ),
                     }
                 )
+        # Call with skipping context to avoid triggering the constraint again
+        dupes_to_unlink.with_context(skip_registration_partner_unique=True).unlink()
```

Comment:
Why doing this here instead on the wizard? This way, it's less readable and non self-contained.

Reply:
Do you mean the unlinking or the whole dupes evaluation?

Reply:
Everything. The constraint shouldn't do anything except constraining.

Reply:
Ok, I retailored the code a little bit to be able to do the attendee cleanup in the wizard